.circleci/config.yml

```diff
@@ -84,7 +84,14 @@ jobs:
               echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$CIRCLE_JDK_VERSION'."
             else
               echo "Deploying snapshot..."
```

Comment:
We should move all this to fastlane, but that's a bigger task and I just want to ship this so we can enjoy this asap